hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -142,6 +142,52 @@ INSTANCE: tb.dut.u_tl_adapter_rom.u_rspfifo
 Condition 7 "1709501387" "(((~gen_normal_fifo.empty)) & ((~gen_normal_fifo.under_rst))) 1 -1"
                            (2 "10")
 
+// The condition !rvalid_o & rready_i is non-occuring. rready_i is reqfifo_rready in
+// u_tl_adapter_rom. reqfifo_rready is driven by d_ack, which is only true if tl_o_int.d_valid is
+// true. Which means that ROM is responding to a request which should be in u_reqfifo as soon as it
+// was out. This implies that u_reqfifo is not empty and both rvaid_o and rready_i must be true.
+INSTANCE: tb.dut.u_tl_adapter_rom.u_reqfifo
+Condition 3 "1324655787" "(rvalid_o & rready_i & ((~gen_normal_fifo.under_rst))) 1 -1" (1 "011")
+
+// If rready_i is true, it means that it is true in u_tl_adapter_rom as rspfifo_rready. If
+// rspfifo_rready is true, this means that reqfifo_rdata.op = OpRead, reqfifo_rdata.error is false
+// and reqfifo_rready is true. The latter signal is driven by d_ack, which can only be true if
+// tl_o_int.d_valid is true. Since we want reqfifo_rdata.op = OpRead and reqfifo_rdata.error as
+// false, the logic that drive d_valid yields rspfifo_rvalid. That signal is rvalid_o in u_rspfifo.
```

Comment:
Grammar nit (sorry): I think you want "the logic that drive**s**" here (because "logic" is singular).

I'd also say something like "yields rspfifo_rfifo=1" (or maybe "means") to make it clear that we're saying the signal is asserted.